### PR TITLE
ORM 2.9 release post

### PR DESCRIPTION
**Pull request by beberlei**

### CODE REVIEW

File: source/blog/2021-05-12-orm2.9.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: "New Release: Doctrine ORM 2.9 with Attributes, Typed Properties, more"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2021/05/12/orm2.9.html
+---
+
+We have released a new minor version Doctrine ORM 2.9, the first version with
```
Comment (SenseException) [inline source/blog/2021-05-12-orm2.9.md:8]:
```suggestion
We have released a new minor version 2.9 of Doctrine ORM, the first version with
```

File: source/blog/2021-05-12-orm2.9.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+ * @template-extends EntityRepository<User>
+ */
+class UserRepository extends EntityRepository
+{
+}
+```
+
+## Query::HINT_READ_ONLY
+
+A new query hint is added that allow hydrating entities through DQL that are
```
Comment (SenseException) [inline source/blog/2021-05-12-orm2.9.md:92]:
```suggestion
A new query hint is added that allows hydrating entities through DQL that are
```

File: source/blog/2021-05-12-orm2.9.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+```
+
+## Deprecations
+
+Doctrine ORM 2.9 rethinks deprecations and integrates with our new
+[doctrine/deprecations](https://github.com/doctrine/deprecations/) library.
+
+- Undeprecate `merge()` and `detach()` as no replacements are available yet
+- Notify Persist Change Tracking: Use Explicit Change Tracking instead
+- DQL `SELECT PARTIAL` syntax, use Value Objects instead
```
Comment (SenseException) [inline source/blog/2021-05-12-orm2.9.md:139]:
```suggestion
- DQL `SELECT PARTIAL` syntax, use Value Objects with `SELECT NEW` instead
```

File: source/blog/2021-05-12-orm2.9.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+- Deprecate `doctrine/cache` for metadata caching, use PSR-6 cache instead
+
+## PHP 7.1 Support
+
+ORM 2.9 reintroduces PHP 7.1 support, because it wasn't technically unsupported
+anyways.  No changes were necessary to the code to allow it again except in the
+testsuite.
+
+The PHP 7.1 support was re-added to allow a very broad approach to prepare for
+some of the deprecations that we are going to prepare making for ORM 3.
```
Comment (SenseException) [inline source/blog/2021-05-12-orm2.9.md:153]:
```suggestion
some of the deprecations that we are going to introduce in ORM 3.
```

**Review by SenseException — CHANGES_REQUESTED** [review]

**Review by SenseException — APPROVED** [review]

File: source/blog/2021-05-24-orm2.9.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+- Deprecate `doctrine/cache` for metadata caching, use PSR-6 cache instead
+
+## PHP 7.1 Support
+
+ORM 2.9 reintroduces PHP 7.1 support, because it wasn't technically unsupported
+anyways.  No changes were necessary to the code to allow it again except in the
+testsuite.
+
+The PHP 7.1 support was re-added to allow a very broad approach to prepare for
+some of the deprecations that we are going to introduce in ORM 3.
```
Comment (alcaeus) [inline source/blog/2021-05-24-orm2.9.md:157]: Are you talking about code that will be deprecated in 3.0, or deprecated in 2.x and dropped in 3.0?

Comment (beberlei) [reply source/blog/2021-05-24-orm2.9.md]: good point!

**Pull request merged**